bots/tests-scan

```diff
@@ -179,13 +81,13 @@ def main():
     opts.repo = api.repo
 
     try:
-        policy = REPO_BRANCH_CONTEXT[opts.repo]
+        policy = testmap.tests_for_project(opts.repo)
         if opts.context:
             policy = {}
-            for branch in REPO_BRANCH_CONTEXT[opts.repo]:
+            for branch in testmap.tests_for_project(opts.repo):
                 branch_context = []
                 for context in opts.context:
-                    if context in REPO_BRANCH_CONTEXT[opts.repo][branch]:
+                    if context in testmap.tests_for_project_branch(opts.repo, branch):
```

Comment:
Let's please avoid that, as above. Instead, iterate over `for branch, branch_contexts in testmap.tests_for_project(opts.repo).items():`